lib/adaptors/serialport.js

```diff
@@ -119,3 +119,7 @@ Adaptor.prototype.onRead = function read(callback) {
 Adaptor.prototype.close = function close(callback) {
   this.serialport.close(callback);
 };
+
+Adaptor.prototype.newSerialport = function() {
+  return new SerialPort(this.conn, {});
+};
```

Comment:
And in this case you could just re-write to stub out `require("thing").Serialport`